addons/l10n_co/models/res_partner.py

```diff
@@ -40,3 +40,16 @@ def _compute_verification_code(self):
                     partner.l10n_co_verification_code = 11 - number
             except ValueError:
                 partner.l10n_co_verification_code = False
+
+    @api.constrains('vat', 'commercial_partner_country_id', 'l10n_co_document_type')
+    def check_vat(self):
+        # check_vat is implemented by base_vat which this localization
+        # doesn't directly depend on. It is however automatically
+        # installed for Colombia.
+        if self.env.ref('base.module_base_vat').state == 'installed':
```

Comment:
We usually do:
``` python
if 'base_vat' in self.env:
```

e.g. https://github.com/odoo/odoo/blob/11.0/addons/crm/wizard/base_partner_merge.py#L392

Reply:
I can only get that to work with models, not modules. And unfortunately `base_vat` doesn't define any models, so I kept it like this. (I did add a `sudo()` to make it work with non-admin users).

Reply:
Yeah indeed, I didn't think about it!